### PR TITLE
Home react plugin

### PR DESCRIPTION
**Pull request by binishma-ss**
Adds the no-top-level-material-ui-4-import ESLint rule to the git-release-manager plugin to aid with the migration to Material UI v5.
Issue: [#23467](https://github.com/backstage/backstage/issues/23467)

### CODE REVIEW

**Comment by AmbrishRamachandiran** [comment]
@awanlin @freben @Rugvip  Please take a Look ! commenting for as there are no reviewers assigned

**Comment by awanlin** [comment]
@AmbrishRamachandiran, the Home plugin is maintained by the `discoverability-maintainers` and they were added to this PR. I'll take a look as well though 👍

**Review by awanlin — APPROVED** [review]
✅ looks good @binishma-ss, thanks for this!

**Comment by awanlin** [comment]
Hi @binishma-ss, just wanted to warn you to rotate your GitHub secrets ASAP as they are still in the commit history of this PR.

**Comment by binishma-ss** [comment]
Waiting for PR to get merged, commenting for avoiding  Stale

**Review by emmaindal — APPROVED** [review]

**Pull request merged**
